### PR TITLE
Fix ABI settings for clang on linux

### PR DESCRIPTION
**Pull request by seelabs**

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -81,6 +81,14 @@ GCC 5: If the gcc toolchain is used, gcc version 5 or better is required. On
     either be built with gcc 4 or with the preprocessor flag
     `_GLIBCXX_USE_CXX11_ABI` set to zero.
 
+Clang on linux: Clang cannot use the new gcc 5 ABI (clang does not know about
+    the `abi_tag` attribute). On linux distros that ship with the gcc 5 ABI
+    (ubuntu >= 15.10), building with clang requires building boost and protobuf
+    with the old ABI (best to build them with clang). It is best to statically
+    link rippled in this scenario (use the `--static` with scons), as dynamic
+    linking may a library with the incorrect ABI.
```
Comment (ximinez) [inline SConstruct]: Missing a verb. "as dynamic linking may **use?/find?** a library with the incorrect ABI".

**Comment by ximinez** [comment]
:+1: Built on Windows (which isn't really relevant), and Ubuntu 15.04 and 14.04 without any hassle.

**Comment by miguelportilla** [comment]
:+1: Built on Windows, Ubuntu 15.04/GCC 5.2, OS X, all tests pass.

**Comment by ximinez** [comment]
Ubuntu 15.04 builds were GCC 5.2.0, and clang 3.6.0. Ubuntu 14.04 builds were GCC 5.1.0, and clang 3.6.0.

**Comment by nbougalis** [comment]
Merged as 41ff751cd261f5625e5561e3dab1ad08e1b26477.

**Pull request closed without merging**